index.js

```diff
@@ -1,17 +1,73 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    let isInserted = false;
+    for (let i = 0; i < this.length; i++) {
+      if (item <= this.items[i]) {
+        this.items.splice(i, 1, item, this.items[i]);
```

Comment:
Muito interessante sua implementação de um sort para arrays numéricos!!! Outra forma possível:

this.items.push(item);
this.items.sort((a, b) => a - b); ====> faz o sort em order crescente ;-)
this.length++;